### DISCUSSION TITLE
painter

### DISCUSSION THREAD

**Discussion by hellersqueen** [AprilSylph/XKit-Rewritten#987] [Help Desk] [answered]
Hello! I'd like to add a colored line to a post when it's been reblogged on my main blog AND also add a colored line when a post has been reblogged but on a sideblog, is this possible? I haven't been able to make it work so far.

**Comment by marcustyphoon** [comment, marked as answer]
I believe Painter can only tell things about the current post it's looking at, unfortunately, e.g. "if this post is a reblog, color it X." We don't have an easy way to tell if a particular post has been reblogged by you before, if I'm guessing correctly that that's what you're trying to do.

**Reply by hellersqueen** [reply]
Oh i see! Yes, that's what I was trying to do but I get now why that doesn't seem to work. Thanks for the quick answer!
